frontend/vue/components/UserAccount/AccountAdmin.vue

```diff
@@ -0,0 +1,289 @@
+      apiToken: ''
+    }
+  },
+  computed: {
+    markdownItInstance (): MarkdownIt { return MarkdownIt(this.options) },
+    privacyPolicyHTML (): string { return this.markdownItInstance.render(this.privacyPolicyMd) }
+  },
+  mounted () {
+    this.privacyPolicyMd = document.getElementById('privacyPolicy')?.textContent || ''
+
+    this.fetchUserToken()
+  },
+  methods: {
+    submitForm () {
+      const token = (this.$refs.apiTokenInput as BXInput).value
+      const csrfToken = { _csrf: window.csrfToken }
+      this.apiToken = token
+      fetch(fetchUrl, {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify({
+          apiToken: token,
+          ...csrfToken
+        })
+      }).then((res) => {
+        if (res.status === 200) {
+          res.json().then((jsonResult) => {
+            console.log(jsonResult, 'result')
+          })
+        } else {
+          // TODO: Manage this error (and improve backend feedback)
+          console.error(res)
+        }
+      })
+    },
+    fetchUserToken () {
+      fetch(fetchUrl, {
+        method: 'GET',
+        headers: { 'Content-Type': 'application/json' }
+      }).then((res) => {
+        if (res.status === 200) {
+          res.json().then((jsonResult) => {
+            this.apiToken = jsonResult.apiToken
+          })
+        } else {
+          // TODO: Manage this error (and improve backend feedback)
+          console.error(res, 'error')
+        }
+      })
+    },
+    updateToken () {
+      this.apiToken = ''
+    }
+  }
+})
+</script>
+
+<style lang="scss" scoped>
+@import 'carbon-components/scss/globals/scss/typography';
+@import '~/../scss/variables/mq.scss';
+@import '~/../scss/variables/colors.scss';
+@import '~/../scss/mixins/mixins.scss';
+
+.account-admin {
+  @include contained();
+  margin-left: 0;
+  padding-top: $spacing-07;
+  padding-bottom: $spacing-07;
+
+  &__section {
+    padding-bottom: $spacing-05;
+    border-bottom: 2px solid $border-color-light-2;
+  }
+
+  &__info {
+    &:not(:last-child) {
+      margin-bottom: $spacing-07;
+    }
+
+    &__label {
+      display: inline-block;
+      margin-bottom: $spacing-03;
+      padding-right: $spacing-08;
+    }
+
+    &__steps {
+      list-style: decimal;
+      margin: 0 0 0 1.5em;
+    }
+  }
+
+  &__input-field {
+
+    &__container {
+      position: relative;
+      margin-right: $spacing-07;
+      width: 100%;
+      max-width: 22rem;
+
+      bx-input {
+        --cds-ui-04: #{$border-color-tertiary};
+        --cds-field-01: #{$background-color-lighter};
+      }
+    }
+  }
+
+  &__input-form {
+    display: flex;
+    align-items: center;
+  }
+
+  &__copy {
+    position: absolute;
+    top: 1.35rem;
+    right: 0.15rem;
+    height: 2.75rem;
+    width: 2.75rem;
+    background-color: $background-color-lighter;
+    position: absolute;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    &:hover {
```

Comment:
Adding a blank line for better readability:

```suggestion

    &:hover {
```